src/ajax.js

```diff
@@ -846,8 +846,9 @@ jQuery.extend( {
 jQuery.each( [ "get", "post" ], function( _i, method ) {
 	jQuery[ method ] = function( url, data, callback, type ) {
 
-		// Shift arguments if data argument was omitted
-		if ( typeof data === "function" ) {
+		// Shift arguments if data argument was omitted.
+		// Handle the null callback placeholder.
+		if ( typeof data === "function" || data === null ) {
```

Comment:
@gibson042 allowing `undefined` here would break the 4-param signature with the second parameter (`data`) being `undefined` which is e.g. used in the `$.getScript` definition. We could add more checks to make it work but I thought in that case it may be better to just allow `null`. `null` is not an allowed `data` value so we don’t have a conflict.

Reply:
I concur.